### PR TITLE
Boxplot added to instructor view

### PR DESCRIPTION
**Pull request by justin0022**
Decided to import [Victory](https://formidable.com/open-source/victory/) for the boxplot. Added a sample into instructor view for now. Label styling may need to change.

### CODE REVIEW

File: src/containers/Instructor.js [hunk 1]
```diff
@@ -35,7 +36,7 @@ const styles = theme => ({
   }
 })
 
-function Instructor (props) {
+function Instructor(props) {
```
Comment (soohyunkim) [inline src/containers/Instructor.js]: maybe undo the space change for consistency

File: src/containers/Instructor.js [hunk 2]
```diff
@@ -181,6 +182,14 @@ function Instructor (props) {
                   </Paper>
                 ))} />
             </> : <Spinner />}
+            <VictoryChart domainPadding={20}>
+              <VictoryBoxPlot boxWidth={20} data={[
```
Comment (soohyunkim) [inline src/containers/Instructor.js]: Will this data be replaced with json later?

File: src/containers/Instructor.js (hunk 1 above)
Comment (soohyunkim) [inline src/containers/Instructor.js]: Maybe undo the deletion of space for consistency

File: src/containers/Instructor.js (hunk 2 above)
Comment (justin0022) [reply src/containers/Instructor.js]: Ya, it'll be replaced with data fed via API in the future. This data's the sample data that Victory had in their docs.

**Pull request merged**